dash/backend/src/modules/policy/entities/compliance-result-map.ts

```diff
@@ -1,47 +1,73 @@
+/**
```

Comment:
The only functional change in this file was to add reason to ComplianceIssue and modify getAllIssues() to return that info.

The rest is comments/formatting